src/main/java/io/cdap/plugin/gcp/spanner/common/SpannerUtil.java

```diff
@@ -116,9 +111,11 @@ public static String convertSchemaToCreateStatement(String tableName, String pri
         case STRING:
           spannerType = "STRING(MAX)";
           break;
+        case INT:
```

Comment:
does this account for all types in ``Schema.Type`` now?

Reply:
The sink supports all primitive types - boolean, string, int, long, float, double, bytes, date and timestamp. However, source schema can only have boolean, string, long, double, bytes, date and timestamp. This is because java int and float are 4 bytes.